frontend/packages/console-shared/src/components/form-utils/FormFooter.tsx

```diff
@@ -98,7 +98,7 @@ const FormFooter: React.FC<FormFooterProps> = ({
               type="button"
               data-test-id="download-button"
               variant={ButtonVariant.secondary}
-              className="pf-v6-c-button--align-right hidden-sm hidden-xs"
+              className="pf-v6-c-button--align-right pf-v6-u-display-none pf-v6-u-display-flex-on-sm"
```

Comment:
I can still see `Download` button on small screen size(I created cluster-bot cluster `launch 4.20.0-0.nightly-2025-07-09-121230,openshift/console#15261 aws`)
<img width="459" height="749" alt="Screenshot 2025-07-11 at 11 36 30 AM" src="https://github.com/user-attachments/assets/9277a0db-fec8-4b96-bb68-5d752fe0f639" />

For comparison on OCP 419
<img width="455" height="742" alt="Screenshot 2025-07-11 at 11 36 49 AM" src="https://github.com/user-attachments/assets/0c71df34-634a-4d35-938b-43089c5fffbb" />